spec/repository_finder_spec.rb

```diff
@@ -77,12 +77,14 @@
     end
 
     context "when git doesn't exist" do
-      before(:each) { @old_path = ENV.delete("PATH").to_s.split(File::PATH_SEPARATOR) }
-      after(:each)  { ENV["PATH"] = @old_path.join(File::PATH_SEPARATOR) }
+      before(:each) do
+        @old_path = ENV["PATH"]
+        ENV["PATH"] = ""
```

Comment:
?

Reply:
The `PATH` is "reset" to an empty string to simulate the scenario where `git` is not installed.. (especially in CI environments)